build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -160,28 +162,47 @@ def convert_directory(directory_path, write_files, allow_partial_conversion,
   if not os.path.isfile(build_file_path):
     return Status.NO_BUILD_FILE
 
+  autogeneration_tag = f"Autogenerated by {repo_relpath(os.path.abspath(__file__))}"
+
+  header = "\n".join(["#" * 80] + [
+      l.ljust(79) + "#" for l in [
+          f"# {autogeneration_tag} from",
+          f"# {rel_build_file_path}",
+          "#",
+          "# Use iree_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary",
+          "# CMake-only content.",
+          "#",
+          f"# To disable autogeneration for this file entirely, delete this header.",
+      ]
+  ] + ["#" * 80])
+
+  preserved_footer = [PRESERVE_TAG + "\n"]
```

Comment:
```suggestion
  preserved_footer = ["\n" + PRESERVE_TAG + "\n"]
```

Please also add a newline before the tag comment, to split it off from the last regular rule in the file.

Current:
```

iree_cc_test(
  NAME
    wait_handle_test
  SRCS
    "wait_handle_test.cc"
  DEPS
    ::wait_handle
    iree::testing::gtest
    iree::testing::gtest_main
)
### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
```

Should be:
```

iree_cc_test(
  NAME
    wait_handle_test
  SRCS
    "wait_handle_test.cc"
  DEPS
    ::wait_handle
    iree::testing::gtest
    iree::testing::gtest_main
)

### BAZEL_TO_CMAKE_PRESERVES_ALL_CONTENT_BELOW_THIS_LINE ###
```

Reply:
Done.